lib/components/AlbumScreen/song_list_tile.dart

```diff
@@ -250,13 +250,13 @@ class _SongListTileState extends ConsumerState<SongListTile>
                           AppLocalizations.of(context)!.placeholderSource),
               id: widget.parentItem?.id ?? "",
               item: widget.parentItem,
-              // we're playing from an album, so we should use the album's LUFS.
-              // album LUFS sometimes end up being simply `0`, but that's not the actual value
-              contextLufs: (widget.isInPlaylist ||
+              // we're playing from an album, so we should use the album's normalization gain.
+              // album normalization gain sometimes ends up being simply `0`, but that's not the actual value
```

Comment:
With the new changes this shouldn't happen anymore ;)

Reply:
nice. and even if it did, it's not an issue, because the gain change would simply be 0, which is the same as not applying a gain change (which was the fallback). It was different when we were still dealing with LUFS, that would've lead to an 18 dB increase in volume 👀